.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - 3.6
+sudo: required
+services:
+  '- docker'
+script:
+  - 'make test-in-container SELECTORS="--distro fedora-26-x86_64.yaml --multispec-selector version=2.4 --multispec-selector variant=upstream"'
```

Comment:
I forgot, can you disable e-mail notifications?

Reply:
Sure, I'll send another PR.